docs/development/decision-records/2024-08-28-Release-Workflow-Refactor/README.md

```diff
@@ -0,0 +1,82 @@
+# Refactor of the Release Workflow
+
+## Decision
+
+Refactor the existing workflows to accommodate the release of bugfix versions as described in [this](../2024-07-18-Improvements-versioning-scheme/README.md) decision record:
+- Remove the releases and release/X.Y.Z branches in favor of using just the main branch and tags.
+- Change the draft-new-release and publish-new-release to enable hotfix releases
+
+## Rationale
+
+As of this decision record creation date, the release of a new version is triggered with the execution of the `draft-new-release` workflow.
+The workflow branches from main and checks out into a `release/*` branch where a new commit to represent the releasable state is created. Afterward a PR is created to merge the release/X.Y.Z branch into `releases`.
```

Comment:
I think it's not really necessary to fully explain the current state of affairs. Better to briefly highlight the pain-points

Reply:
I'll be brief next time.